### PR TITLE
reply about deep time studies

### PR DESCRIPTION
**Pull request by sdtaylor**
added 2 references and changed the intro wording a bit.

### CODE REVIEW

**Comment by davharris** [comment]
An alternative: https://github.com/weecology/bbs-forecasting/commit/c1a3830eeee66dcd71c30905fc05fff4803fd179?short_path=2112e0d#diff-2112e0db2b5026e7808dfd94ee62b151

File: manuscript/draft.Rmd [hunk 1]
```diff
@@ -99,20 +99,20 @@ of biodiversity [@jetz2007; @thuiller2011], little is known about how
...
-predictive accuracy, and for understanding how well different methods can
-predict the future. These "best practices" for effective forecasting and
-evaluation (Box 1) broadly involve: 1) expanding the use of data to include
-biological and environmental time-series [@tredennick2016]; 2) accounting for
-uncertainty in observations and processes, [@harris2015; @yu2010]; and 3)
-conducting meaningful evaluations of the forecasts by hindcasting, archiving
-short-term forecasts, and comparing forecasts to baselines to determine whether
-the forecasts are more accurate than assuming the system is basically static
-[@perretti2013].
+What we do know comes from a small number of biodiversity studies, where models
```
Comment (ethanwhite) [inline manuscript/draft.Rmd]: Do we need to drop hindcasting here? I think it's clearer to keep it and based on our definition ("any prediction of observations already available is generally a hindcast") the Blois paper at least still works.

File: manuscript/draft.Rmd [hunk 2]
```diff
@@ -99,20 +99,20 @@ of biodiversity [@jetz2007; @thuiller2011], little is known about how
...
+are built using data on species occurrence, richness, or community composition
+from distinct timeperiods and evaluated on their ability to predict contemporary
```
Comment (ethanwhite) [inline manuscript/draft.Rmd]: How about "from different time periods and evaluated on their ability to predict time patterns in contemporary [cites] or historic [cites] periods not used for model fitting."

**Comment by sdtaylor** [comment]
@davharris that looks like an alternative to #227, of which it does look better.

**Comment by davharris** [comment]
@sdtaylor Yeah, I had just moved it.  Sorry!  Should we close #227?

**Comment by ethanwhite** [comment]
:+1: to replacing #227 with @davharris's alternative.

File: manuscript/draft.Rmd (hunk 2 above)
Comment (sdtaylor) [reply manuscript/draft.Rmd]: what do you mean by "time patterns" ?

File: manuscript/draft.Rmd (hunk 1 above)
Comment (sdtaylor) [reply manuscript/draft.Rmd]: Good point, I'll put it back in

File: manuscript/draft.Rmd (hunk 2 above)
Comment (ethanwhite) [reply manuscript/draft.Rmd]: Sorry, didn't manage to get it cleaned up properly:

"from different time periods are evaluated on their predictions for contemporary [cites] or historic [cites] periods not used for model fitting."

Comment (sdtaylor) [reply manuscript/draft.Rmd]: How about:  

What we do know comes from a small number of hindcasting studies, where models
are built from different time periods and evaluated on their ability to predict biodiversity patterns in contemporary [@algar2009; @distler2015] or historic [@maguire2016; @blois2013] periods not used for model fitting.

Comment (ethanwhite) [reply manuscript/draft.Rmd]: :+1:

**Comment by ethanwhite** [comment]
LGTM

**Comment by davharris** [comment]
Thanks both!

**Pull request merged**